src/xenia/kernel/user_module.h

```diff
@@ -38,6 +38,7 @@ class UserModule : public XModule {
 
   const std::string& path() const override { return path_; }
   const std::string& name() const override { return name_; }
+  const uint64_t hash() const { return hash_; }
```

Comment:
The return value doesn't need `const`-ness, just the method.

`uint64_t hash() const { return hash_; }`